oio/cli/directory/directory.py

```diff
@@ -171,24 +171,39 @@ def take_action(self, parsed_args):
 
 
 class DirectoryDecommission(DirectoryCmd):
-    """Decommission a Meta1 service (or only some bases)."""
+    """
+    Decommission a Meta1 service (or only some bases).
+    """
 
     def get_parser(self, prog_name):
         parser = super(DirectoryDecommission, self).get_parser(prog_name)
         parser.add_argument('addr', metavar='<ADDR>',
                             help='Address of service to decommission')
         parser.add_argument('base', metavar='<BASE>', nargs='*',
                             help="Name of bases to decommission")
+        parser.add_argument('--no-check', action='store_true',
+                            help=("Do not check that all prefixes have "
```

Comment:
Minor: at l.199, we still check_replicas so IMO help should be more `Ignore error on missing prefixes' replicas`